pages/blogs/search.js

```diff
@@ -7,28 +7,27 @@ import matter from 'gray-matter';
 import ContentBox from '@components/ContentBox';
 import Search from '@components/Search';
 
-export default ({ allMdx }) => {
+export default function SearchPage({ allMdx }) {
   const [filteredBlogs, setFilteredBlogs] = React.useState(allMdx);
 
   const handleFilter = (data) => {
     setFilteredBlogs(data);
   };
 
   return (
-    <Flex>
+    <Box pb={3}>
       {/* Content Area + Input + Tag filter */}
-      <Flex direction="column" justify="center" alignItems="center" w="100%">
+      <Stack spacing={[4, 8, 12]} justify="center" alignItems="center">
         <Search blogs={allMdx} handleFilter={handleFilter} />
-        <Flex direction="column" justify="space-evenly" h="80vh">
-          {filteredBlogs &&
-            filteredBlogs.map((blog) => (
-              <ContentBox key={blog.slug} blog={blog} />
-            ))}
-        </Flex>
-      </Flex>
-    </Flex>
+        <Stack spacing={[2, 6, 12]}>
```

Comment:
Responsive spacing makes this look crispier on mobile, which leaving plenty of whitespace on larger screens.